game_event_player/tests.py

```diff
@@ -0,0 +1,172 @@
+    player = Player.objects.create(
+        user=user2,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def saved_player3(user3):
+    player = Player.objects.create(
+        user=user3,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def players(saved_player, saved_player2, saved_player3):
+    return tuple([saved_player, saved_player2, saved_player3])
+
+
+@pytest.fixture
+def saved_court():
+    court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                 city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+    return court
+
+
+@pytest.fixture
+def saved_game_event(saved_court):
+    game_event = GameEvent.objects.create(id=ID1, time=datetime.today(), level_of_game=3,
+                                          min_number_of_players=2, max_number_of_players=5, court_id=saved_court,
+                                          ball_game='Basketball')
+    return game_event
+
+
+@pytest.fixture
+def saved_game_event2(saved_court):
+    game_event = GameEvent.objects.create(id=ID2, time=datetime.today(), level_of_game=3,
+                                          min_number_of_players=2, max_number_of_players=5, court_id=saved_court,
+                                          ball_game='Basketball')
+    return game_event
+
+
+@pytest.fixture
+def saved_game_event_player(saved_game_event, saved_player):
+    return GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                          ball_responsible=False)
+
+
+@pytest.fixture
+def create_game_event_players(saved_game_event, saved_players, size):
+    players = []
+    for i in range(size):
+        players.append(GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_players[i],
+                                                      ball_responsible=False))
+    return players
+
+
+@pytest.mark.django_db
+class TestGameEventPlayerModel:
+
+    def test_game_event_player_creation(self, saved_game_event, saved_player):
+        game_event_player = GameEventPlayer.objects.create(game_event_id=saved_game_event,
+                                                           player_id=saved_player,)
+        assert game_event_player.game_event_id == saved_game_event
+        assert game_event_player.player_id == saved_player
+        assert game_event_player.ball_responsible is False
+
+    def test_game_event_player_ball_responsible_default_value(self, saved_game_event, saved_player):
+        game_event_player = GameEventPlayer.objects.create(game_event_id=saved_game_event,
+                                                           player_id=saved_player,).full_clean()
+        assert game_event_player is None
+
+    def test_filter_players_by_event(self, saved_game_event, saved_game_event2, players):
```

Comment:
This test may not be entirely self-explanatory. To improve readability, consider dividing it into three parts and encapsulating each part within a separate function:

1. function for creating GameEventPlayers.
2. a function that will test whether a player is in the expected events.
3. a function that will test whether an event has the expected players.

Breaking down the test into smaller, focused functions will make it easier to read and understand.

Reply:
It's not the prettiest of tests, but we're already over the time we agreed we'll merge stuff. I suggest opening an issue for cleaning this up.

Reply:
Opened an issue related to this #52